src/CMakeLists.txt

```diff
@@ -122,44 +122,37 @@ set(GISH_H
     config.h
 )
 
-if(WIN32)
-    list(APPEND GISH_H video/glext.h)
-endif(WIN32)
-
 if(APPLE)
     list(APPEND GISH_SRCS sdl/SDLMain.m)
     list(APPEND GISH_H sdl/SDLMain.h)
 endif(APPLE)
 
-if(WIN32)
-    add_executable(freegish WIN32 ${GISH_SRCS} ${GISH_H})
-else()
-    add_executable(freegish ${GISH_SRCS} ${GISH_H})
-endif()
+add_executable(freegish ${GISH_SRCS} ${GISH_H})
+
+target_include_directories(freegish PRIVATE
+    ${SDL2_INCLUDE_DIRS}
+    ${OPENAL_INCLUDE_DIR}
+    ${OPENGL_INCLUDE_DIR}
+    ${Vorbis_INCLUDE_DIR}
+    ${PNG_INCLUDE_DIR}
+)
+
+target_link_libraries(freegish
+    ${SDL2_LIBRARIES}
+    ${OPENAL_LIBRARY}
+    ${OPENGL_LIBRARIES}
+    ${Vorbis_LIBRARIES}
+    ${PNG_LIBRARIES}
+)
+
+if(LIBM)
+  target_link_libraries(freegish m)
+endif(LIBM)
+
+set_property(TARGET freegish PROPERTY VS_DEBUGGER_WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}")
 
-if(MSVC)
-    target_link_libraries(freegish PRIVATE
-        ${OPENGL_LIBRARIES}
-        SDL2::SDL2
-        SDL2::SDL2main
-        Ogg::ogg
-        Vorbis::vorbis
-        Vorbis::vorbisenc
-        Vorbis::vorbisfile
-        png
-        OpenAL::OpenAL
-    )
-else()
-    target_link_libraries(freegish
-        ${SDL2_LIBRARIES}
-        ${OPENAL_LIBRARY}
-        ${OPENGL_LIBRARIES}
-        ${Vorbis_LIBRARIES}
-        ${PNG_LIBRARIES}
-        ${M_LIBRARIES}
-    )
+if(INSTALL_FHS)
+  target_compile_definitions(freegish PRIVATE DATAPATH=${FREEGISH_INSTALL_DATA_DIR})
```

Comment:
This results into build errors for me on Ubuntu 22.04:
```
[ 98%] Building C object src/CMakeFiles/freegish.dir/main.c.o
/home/wouter/freegish/src/main.c: In function ‘main’:
/home/wouter/freegish/src/main.c:53:3: error: too few arguments to function ‘chdir’
   53 |   chdir(DATAPATH);
      |   ^~~~~
In file included from /home/wouter/freegish/src/main.c:43:
/usr/include/unistd.h:517:12: note: declared here
  517 | extern int chdir (const char *__path) __THROW __nonnull ((1)) __wur;
      |            ^~~~~
make[2]: *** [src/CMakeFiles/freegish.dir/build.make:860: src/CMakeFiles/freegish.dir/main.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:98: src/CMakeFiles/freegish.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

Reply:
Indeed, I was missing quotes :sweat_smile: Should be fixed now.